labelbox/schema/model_run.py

```diff
@@ -122,6 +125,34 @@ def _wait_until_done(self, status_fn, timeout_seconds=120, sleep_time=5):
                 )
             time.sleep(sleep_time)
 
+    def upsert_predictions_and_send_to_project(
+        self,
+        name: str,
+        predictions: LabelList,
+        project_id: str,
+        priority: Optional[int] = 5,
+    ) -> 'MEAPredictionImport':  # type: ignore
+        """ Upload predictions and creates a batch import to project.
+        Args:
+            name (str): name of the AnnotationImport job as well as the name of the batch import
+            predictions (Iterable):
+                iterable of annotation rows
+            project_id (str): id of the project to import into
+            priority (int): priority of the job
+        Returns:
+            (AnnotationImport, Project)
+        """
+        data_rows_set = set(
+            map(lambda x: x.data_row.id, predictions)[:DATAROWS_IMPORT_LIMIT])
```

Comment:
you can't slice the result of a map function. It returns an iterable. So you can use itertools.islice.